setup.py

```diff
@@ -34,6 +25,8 @@
         'Programming Language :: Cython',
         'Programming Language :: Python :: 2',
         'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.5',
```

Comment:
I have tested only on 3.6, so I'll put 3.6 here and in tox.ini too, as sole py3 versions - that way we keep latest versions only. If you think previous ones should also be included/tested, I'll add them too.